worlds/shivers/Options.py

```diff
@@ -24,49 +26,58 @@ class LobbyAccess(Choice):
     option_local = 2
     default = 1
 
+
 class PuzzleHintsRequired(DefaultOnToggle):
     """
     If turned on puzzle hints/solutions will be available before the corresponding puzzle is required.
 
-    For example: The Red Door puzzle will be logically required only after access to the Beth's Address Book which gives you the solution.
+    For example: The Red Door puzzle will be logically required only after access to the Beth's Address Book
```

Comment:
Should be able to remove this word. You could also change it to "getting access" or "obtaining access"
```suggestion
    For example: The Red Door puzzle will be logically required only after access to Beth's Address Book
```